turbopack/crates/turbopack-ecmascript/src/analyzer/builtin.rs

```diff
@@ -97,6 +97,23 @@ pub fn early_replace_builtin(value: &mut JsValue) -> bool {
 /// processed.
 pub fn replace_builtin(value: &mut JsValue) -> bool {
     match value {
+        JsValue::Add(_, list) => {
+            if list.iter().all(|arg| arg.is_string() == Some(false)) {
```

Comment:
That check seem weird and unnecessary. It should be handled by the `let JsValue::Constant(ConstantValue::Num(num)) = arg else` already